.github/workflows/linux-cpu-arm64-build.yml

```diff
@@ -95,4 +95,4 @@ jobs:
         run: |
           docker run --rm \
           --volume $GITHUB_WORKSPACE:/onnxruntime_src \
-          -w /onnxruntime_src ort_genai_linux_arm64_gha bash -c "/onnxruntime_src/build/cpu/test/unit_tests"   
+          -w /onnxruntime_src ort_genai_linux_arm64_gha bash -c "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/onnxruntime_src/build/cpu/ /onnxruntime_src/build/cpu/test/unit_tests"
```

Comment:
Unlike Python wheel, we can't reliably determine where to find ORT for native C++ binaries like `unit_tests`. Fortunately , `dlopen` also respect `LD_LIBRARY_PATH`.